layers/meta-balena-5x-owa/recipes-support/resin-init/files/resin-init-board

```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+LOCKFILE="/var/lock/owaapi.lck"                                                            
+
+sleep 10
+while [ -f "$LOCKFILE" ]
+do
+    sleep 1
+done
+
 /usr/bin/Start_BT_WiFi 1
```

Comment:
Could you update the microcontroller firmware with a hostOS update hook? Similar to how we update the bootloader for example https://github.com/balena-os/balena-variscite/tree/master/layers/meta-balena-imx6ul-var-dart/recipes-support/hostapp-update-hooks

Reply:
The highlighted change is not forcing a microcontroller flashing, but waiting for it to be flahsed. Our powermanager service ( pmsrv ) will check the FW version of the uC and flash it in case it is older than the one it is looking for. 
The `/var/lock/owaapi.lck`file will be present until that automatic uC flashing ends, thus, the loop of the commit is just a mean to be sure we are not trying to communicate the uC ( which enables BT, WIFI, GSM modules... ) before it is correctly flashed and running.

Reply:
After reviewing the hooks option I think we prefer to go with this PR insted, but thanks for the suggestion! I'll try to leverage that hooks in further modifications.